src/vivarium/framework/results/context.py

```diff
@@ -144,7 +144,10 @@ def gather_results(
                     aggregator = observation.aggregator
                     aggregates = self._aggregate(pop_groups, aggregator_sources, aggregator)
                     aggregates = self._coerce_to_dataframe(aggregates)
-                    aggregates.rename(columns={aggregates.columns[0]: "value"}, inplace=True)
+                    if aggregates.shape[1] == 1:
```

Comment:
Should we move this inside `_coerce_to_dataframe`? We also talked about renaming `_coerce_to_dataframe`.